modules/grafana/templates/dashboards/publishing_api_overview.json.erb

```diff
@@ -829,6 +829,91 @@
         }
       ],
       "title": "New row"
+    },
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "Graphite",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {
+            "threshold1": null,
```

Comment:
Puppet templates are ruby ERB files under the hood. You'd probably want something like this

```<%
  redis_max_memory = (@memorysize_mb / 4 * 3).floor

  redis_mem_warn = $redis_max_memory * 0.8
  redis_mem_crit = $redis_max_memory * 0.9
%>

Literal string = <%= redis_mem_warn %>
```

Which is a duplication of the code that works out the values but is also much easier than trying to get the values from redis to a graphing host